docs/config/README.md

```diff
@@ -605,6 +607,23 @@ The module will be shown if any of the following conditions are met:
 [golang]
 symbol = "🏎💨 "
 ```
+## Haskell
+
+The `haskell` module shows the currently installed version of Haskell Stack version.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `stack.yaml` file
+
+### Options
+
+| Variable   | Default       | Description                                               |
+| ---------- | ------------- | --------------------------------------------------------- |
+| `symbol`   | `"λ "`        | The symbol used before displaying the version of Haskell. |
+| `style`    | `"bold red"`  | The style for the module.                                 |
+| `disabled` | `false`       | Disables the `haskell` module.                            |
+
```

Comment:
It appears you're missing the "Example" heading that other modules have:

```
### Example

```toml
# ~/.config/starship.toml

[haskell]
symbol = "🍕 "
```

Reply:
Missed a bunch of stuff when rebasing and fixing conflicts. 🤦‍♂